layouts/_default/baseof.html

```diff
@@ -21,9 +21,8 @@
 
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather:300,700%7CSource+Code+Pro:400,700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-awesome@1.1.7/css/fork-awesome.min.css"
-      integrity="sha256-gsmEoJAws/Kd3CjuOQzLie5Q3yshhvmo7YNtBG7aaEY=" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/normalize.css@8/normalize.min.css">
+    <link rel="preload" href="/fonts/forkawesome-webfont.woff2?v=1.1.7" as="font" type="font/woff2" crossorigin>
```

Comment:
Is this line really necessary? The CSS code should load the font files for us.
Maybe this is related with the path used for loading the fonts? `src: url('../fonts/forkawesome-webfont.eot?v=1.1.7');` Notice that there is a `..` here, maybe the fonts are not being found.

Reply:
this essentially helps with pagespeed/lighthouse score, since css files are being loaded a bit later, the ../ path is fine in the css files and it works even without this particular line of course :)

Reply:
I'd say so, browsers known to implement WOFF 2.0 include:

Google Chrome (36+),
Chrome for Android(36+),
Opera (24+),
Samsung Internet (4+),
QQ Browser (1.2+) (all Blink-based)
Mozilla Firefox (39+)
Microsoft Edge (14+)
Safari (10+, on Sierra or later)
iOS safari (10.2+)